### PR TITLE
Add multilang data to resolveMenuItem() methods

### PR DESCRIPTION
**Pull request by mjauvin**
The plugin now handles its own multilang functionality.

See:
https://github.com/wintercms/wn-sitemap-plugin/pull/11
https://github.com/wintercms/wn-translate-plugin/pull/57

### CODE REVIEW

File: models/Category.php [hunk 1]
```diff
@@ -301,8 +399,7 @@ protected static function getCategoryPageUrl($pageCode, $category, $theme)
         }
 
         $paramName = substr(trim($matches[1]), 1);
-        $url = CmsPage::url($page->getBaseFileName(), [$paramName => $category->slug]);
 
-        return $url;
+        return [ $paramName => $category->slug ];
```
Comment (LukeTowers) [inline models/Category.php:403]:
```suggestion
        return [$paramName => $category->slug];
```

**Comment by mjauvin** [comment]
Tested with all Blog menuItem types:

- Blog Post
- Blog Category Posts
- All Blog Posts
- Blog Category
- All Blog Categories

With slugs for both locales ['en', 'fr'] (en is default)

**Comment by LukeTowers** [comment]
@mjauvin are you able to retest with my latest changes? Also if you could review and let me know your thoughts that would be appreciated :)

**Comment by mjauvin** [comment]
> @mjauvin are you able to retest with my latest changes? Also if you could review and let me know your thoughts that would be appreciated :)

will do

**Comment by LukeTowers** [comment]
@mjauvin is this working to your satisfaction? Should I move on to the other PRs now?

**Comment by mjauvin** [comment]
> @mjauvin is this working to your satisfaction? Should I move on to the other PRs now?

Yes, I marked as status:completed.

**Comment by mjauvin** [comment]
Can we merge this?

**Pull request merged**